linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_delete/ansible/shared.yml

```diff
@@ -4,7 +4,7 @@
 # disruption = low
 # strategy = configure
 
-{{% if product in ["ol8", "rhel8"] %}}
+{{% if product in ["ol8"] or 'rhel' in product %}}
```

Comment:
And what about RHEL 9? Is this change expected also for RHEL 9 or should it be only for RHEL 7 and 8?

Reply:
From what I understand we are trying to keep RHEL9 aligned and up to date with other RHELs.
@Mab879 Thoughts on this change?

Reply:
If this truly works on RHEL7+, we can keep it this way as it may help reduce work later. The way this was written makes me think it's RHEL8+ only.

Reply:
Well, we suggested this change to RHEL7 STIG content, :D.
And the `auid` filters are pretty much universal, we have similar filters for other audit rules applicable to RHEL7:
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_successful_file_modification_creat/rule.yml#L14

Reply:
@jan-cerny @Mab879 I don't think the `auid` filters would cause problems for RHEL-9.

@ggbecker Any thoughts?

Reply:
OK, thanks for clarification.

Reply:
It should be fine for RHEL9 as well.

Reply:
Seems fine to me then.